server/views/pages/recommendations/selectIndexOffence.njk

```diff
@@ -21,7 +21,7 @@
 
             {% if errorMessage %}
                 {% set html %}
-                    <h1 class="govuk-heading-m" data-qa='multiple-results'>NOMIS error</h1>
+                    <h1 class="govuk-heading-m" data-qa='multiple-results'>NOMIS Error</h1>
```

Comment:
Beck has been making the point that titles are not camel case.

Reply:
'Error' should be 'error'

Reply:
corrected